sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -78,6 +80,7 @@ object SQLMetrics {
   private val SUM_METRIC = "sum"
   private val SIZE_METRIC = "size"
   private val TIMING_METRIC = "timing"
+  private val NORMALIZE_TIMING_METRIC = "normalizeTiming"
```

Comment:
maybe `nsToMsTiming`?

Reply:
Actually I think your previous naming is good, sorry for the back and forth.

Reply:
`private val NS_TIMING_METRIC = "nsTiming"`

Reply:
Thanks, there's no back and forth, thanks for your advise and help all along Wenchen. :)